webots_ros2_tutorials/webots_ros2_tutorials/slave.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 1996-2020 Soft_illusion.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import rclpy
+from webots_ros2_core.webots_node import WebotsNode
+from std_msgs.msg import Float64
+from geometry_msgs.msg import Twist
+
+
+class ServiceNodeVelocity(WebotsNode):
+    def __init__(self, args):
+        super().__init__('slave_node', args)
+
+        # Enable 3 sensors
+        self.service_node_vel_timestep = 16
+
+        # Sensor section
+        self.sensorTimer = self.create_timer(
+            0.001 * self.service_node_vel_timestep, self.sensor_callback)
+
+        self.right_sensor = self.robot.getDistanceSensor(
+            'distance_sensor_right')
+        self.right_sensor.enable(self.service_node_vel_timestep)
+        self.sensorPublisher_right = self.create_publisher(
+            Float64, 'right_IR', 1)
+
+        self.mid_sensor = self.robot.getDistanceSensor('distance_sensor_mid')
+        self.mid_sensor.enable(self.service_node_vel_timestep)
+        self.sensorPublisher_mid = self.create_publisher(Float64, 'mid_IR', 1)
+
+        self.left_sensor = self.robot.getDistanceSensor('distance_sensor_left')
+        self.left_sensor.enable(self.service_node_vel_timestep)
+        self.sensorPublisher_left = self.create_publisher(
+            Float64, 'left_IR', 1)
+
+        self.get_logger().info('Sensor enabled')
+
+        # Front wheels
+        self.leftMotor_front = self.robot.getMotor('left_front_wheel')
+        self.leftMotor_front.setPosition(float('inf'))
+        self.leftMotor_front.setVelocity(0)
+
+        self.rightMotor_front = self.robot.getMotor('right_front_wheel')
+        self.rightMotor_front.setPosition(float('inf'))
+        self.rightMotor_front.setVelocity(0)
+
+        # Rear wheels
+        self.leftMotor_rear = self.robot.getMotor('left_rear_wheel')
+        self.leftMotor_rear.setPosition(float('inf'))
+        self.leftMotor_rear.setVelocity(0)
+
+        self.rightMotor_rear = self.robot.getMotor('right_rear_wheel')
```

Comment:
Should `self.right_motor_rear`. It seems you missed to fix notation in this file

Reply:
I checked full file. Now there is not variable with capital letter in slave.py